tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -11,10 +12,40 @@
 from marionette import MarionetteTestCase, B2GTestCaseMixin
 from marionette_driver import expected, By, Wait
 from marionette_driver.errors import NoSuchElementException, StaleElementException
+import mozfile
 
 from environment import GaiaTestEnvironment
 from file_manager import GaiaDeviceFileManager, GaiaLocalFileManager
 
+DEFAULT_SETTINGS = {
```

Comment:
I'm not sure of the pertinence of the comments here. Most of them are just repeating the key and the associated value. 
I would remove them, except the ones that add more context.